kernel/runlevel/3_distribution/rsync/rsync_windows.go

```diff
@@ -0,0 +1,49 @@
+/*
+	Copyright 2019 NetFoundry, Inc.
+
+	Licensed under the Apache License, Version 2.0 (the "License");
+	you may not use this file except in compliance with the License.
+	You may obtain a copy of the License at
+
+	https://www.apache.org/licenses/LICENSE-2.0
+
+	Unless required by applicable law or agreed to in writing, software
+	distributed under the License is distributed on an "AS IS" BASIS,
+	WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+	See the License for the specific language governing permissions and
+	limitations under the License.
+*/
+
+package rsync
+
+import (
+	"fmt"
+	"github.com/netfoundry/fablab/kernel/internal"
+	zitilab_bootstrap "github.com/netfoundry/fablab/zitilab/development/bootstrap"
+	"github.com/sirupsen/logrus"
+	"strings"
+)
+
+func rsync(sourcePath, targetPath string) error {
+	//Only Cygwin's OpenSSH ssh binary and Cygwin's rsync binary used together seem to work.
+	//Using cwRsync + Microsoft's OpenSSH ssh port did not work 1st quarter 2020.
+	if !strings.Contains(zitilab_bootstrap.RsyncCommand(), "cygwin") {
```

Comment:
Should `RsyncCommand` be in `zitilab_bootstrap`? This seems like it should be in `kernel`. Same with `SshCommand` below.